dependabot-auto-merge/action.yml

```diff
@@ -9,22 +8,5 @@ inputs:
     required: true
 
 runs:
-  using: "composite"
-  steps:
-    - name: Dependabot metadata
-      id: dependabot-metadata
-      uses: dependabot/fetch-metadata@v1
-    - name: Enable auto-merge for Dependabot PRs
-      shell: bash
-      if: ${{ contains(fromJson('["version-update:semver-minor", "version-update:semver-patch"]'), steps.dependabot-metadata.outputs.update-type) && github.event.pull_request.user.login == 'dependabot[bot]' }}
-      run: gh pr merge --auto --merge "$PR_URL"
-      env:
-        PR_URL: ${{github.event.pull_request.html_url}}
-        GITHUB_TOKEN: ${{inputs.GITHUB_TOKEN}}
-    - name: Approve a PR
-      shell: bash
-      if: ${{ contains(fromJson('["version-update:semver-minor", "version-update:semver-patch"]'), steps.dependabot-metadata.outputs.update-type) && github.event.pull_request.user.login == 'dependabot[bot]' }}
-      run: gh pr review --approve "$PR_URL"
-      env:
-        PR_URL: ${{github.event.pull_request.html_url}}
-        GITHUB_TOKEN: ${{inputs.GITHUB_TOKEN}}
+  using: 'node16' # GitHub only supports values listed here: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsusing-for-javascript-actions
```

Comment:
I tested this! https://github.com/Automattic/vip-actions/pull/18#discussion_r1100973372